cmd/sync.go

```diff
@@ -65,6 +65,8 @@ type rawSyncCmdArgs struct {
 	deleteDestination string
 
 	s2sPreserveAccessTier bool
+
+	blobTags string
```

Comment:
I don't know how I feel about blob tags on sync.

It would make sense if it were a way to filter the source. But, in this case, it looks like it's not. Which feels against the spirit of sync.

Reply:
I would normally advise against adding info for dest that src didn't have, but I just realized that tags might be a special case.

If we allow tags in sync, then the user can tag blobs that get copied with a specific value, thus allowing an interesting scenario: if they want, they can distinguish the batches of blobs that got moved over, and do processing on them later. I have no idea if this is an actual scenario for cx, but it sounds feasible.